tools/map/build_external_places.py

```
@@ -0,0 +1,279 @@
+    ("一大國", "EXTERNAL_PLACE", 4, "Iki", JP, None, None,
+     "東夷傳 對馬國 남쪽 천여 리 瀚海. 이키섬 하루노쓰지 유적", "Nagasaki Prefecture", 20),
+    ("末盧國", "EXTERNAL_PLACE", 4, "Karatsu", JP, None, None,
+     "東夷傳 「又渡一海，千餘里至末盧國，有四千餘戶，濵山海居」. 가라쓰", None, 20),
+    ("伊都國", "EXTERNAL_PLACE", 4, "Itoshima", JP, None, None,
+     "東夷傳 「東南陸行五百里，到伊都國」. 이토시마 미쿠모 유적", None, 20),
+    ("奴國", "EXTERNAL_PLACE", 4, "Kasuga", JP, None, None,
+     "後漢書 「建武中元二年，倭奴國奉貢朝賀 … 倭國之極南界也」. 金印은 志賀島 출토지만 "
+     "왕도 후보는 須玖岡本 유적의 가스가", "Fukuoka Prefecture", 20),
+    ("邪馬壹國", "EXTERNAL_PLACE", 6, "Yoshinogari", JP, None, None,
+     "三國志는 邪馬壹國, 後漢書는 邪馬臺國으로 적는다(원문에 壹/臺 이체자 주석). "
+     "규슈설 채택 — 기나이설(나라 분지)과 미결", None, 15),
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)